lib/smart_proxy_dns_powerdns/dns_powerdns_main.rb

```diff
@@ -59,7 +59,7 @@ def create_ptr_record(fqdn, ptr)
 
     def do_create(name, value, type)
       zone = get_zone(name)
-      unless create_record(zone['id'], name, type, value) and rectify_zone(zone['name'])
+      unless create_record(zone['id'], name, type, value) and update_soa(zone['id'], zone['name']) and rectify_zone(zone['name'])
```

Comment:
That may be an option. The only problem here is that I do not know how the REST backend behaves. Maybe, most of those post-save functions would be empty for the REST backend. Do you have any insights on that?